app/models/user.rb

```diff
@@ -45,6 +45,13 @@ def full_name
     [first_name, last_name].compact_blank.join(' ')
   end
 
+  # used by the (devise) mailers
+  def name_for_email
```

Comment:
is this intended to be `name_or_email` ?

Looking at Katjas suggestion in the issue I think maybe this should return an empty string but would need to sort spacing with the `Hello` and the `,`

> Hello Dave Daverson,

vs

> Hello,